modal/cli/config.py

```diff
@@ -17,10 +16,15 @@
 )
 
 
-@config_cli.command(help="Show configuration values for the current profile (debug command).")
-def show():
+@config_cli.command(help="Show current configuration values (debugging command).")
+def show(redact: bool = typer.Option(True, help="Redact the `token_secret` value.")):
     # This is just a test command
-    pprint.pprint(config.to_dict())
+    config_dict = config.to_dict()
+    if redact and config_dict.get("token_secret"):
+        config_dict["token_secret"] = "..."
```

Comment:
Using `***` seems to be a lot more common than `...`.

Reply:
Oh yeah good idea